code/index.html

```diff
@@ -1,22 +1,106 @@
 <!DOCTYPE html>
 <html>
+
   <head>
+
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+
   </head>
+
   <body>
+
+    <div>
+
+      <img class="header-image" src="background-image.jpg" alt="header-image" >
+
+    </div>
+
     <header>
-      <!-- Some kind of logo and navbar -->
+
+      <div class="site-name">
+
+      <img src="hiclipart.com.png" class="img-logo" alt="company-logo"> 
+
+      <h1>Techtrends</h1>
+
+      </div>
+      
+      <div class="desktop-menu-navbar">
+
+      <span class="navbar-link"><a class="link-desktop-navbar" href="" alt="link" target="_blank">News</a></span>
+      <span class="navbar-link"><a class="link-desktop-navbar" href="" alt="link" target="_blank">Sports</a></span>
+      <div class="site-name-tablet">
+        <img src="hiclipart.com.png" class="img-logo" alt="company-logo"> 
```

Comment:
Alt text is written like "normal" text
```suggestion
        <img src="hiclipart.com.png" class="img-logo" alt="company logo"> 
```